app/src/main/java/org/sopt/at/MyActivity.kt

```diff
@@ -0,0 +1,128 @@
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.at.ui.theme.ATSOPTANDROIDTheme
+
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val userId = intent.getStringExtra("id") ?: "id"
+        enableEdgeToEdge()
+        setContent {
+            ATSOPTANDROIDTheme  {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    MyScreen(modifier =Modifier.padding(innerPadding),userId=userId)
+
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun Greeting4(name: String, modifier: Modifier = Modifier) {
+    Text(
+        text = "Hello $name!",
+        modifier = modifier
+    )
+}
+
+@Preview(showBackground = true)
+@Composable
+fun GreetingPreview4() {
+    ATSOPTANDROIDTheme  {
+        MyScreen(userId = "id")
+    }
+}
+
+@Composable
+fun MyScreen(modifier: Modifier = Modifier, userId: String) {
+    val context = LocalContext.current
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
+            .padding(20.dp),
+        verticalArrangement = Arrangement.SpaceBetween
+    ) {
+        Column(
+            modifier = Modifier
+                .padding(top = 16.dp)
+        ) {
+            Text(
+                text = "프로필 정보",
+                color = Color.White,
+                fontSize = 28.sp,
+
+                )
+
+            Text(
+                modifier = Modifier
+                    .padding(top = 24.dp),
+                text = "안녕하세요, $userId 님!",
+                color = Color.White,
+                fontSize = 24.sp
+            )
+
+            Text(
+                modifier = Modifier
+                    .padding(top = 8.dp),
+                text = "환영합니다.",
+                color = Color.LightGray,
+                fontSize = 16.sp
+            )
+        }
+
+        Button(
+            onClick = {
+                val sharedPref = context.getSharedPreferences("auth", Context.MODE_PRIVATE)
+                with(sharedPref.edit()) {
+                    putBoolean("isLoggedIn", false)
+                    apply()
+                }
+
+                val intent = Intent(context, SigninActivity::class.java)
+                intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+                context.startActivity(intent)
+                (context as? Activity)?.finish()
+            },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(20.dp),
+            shape = RoundedCornerShape(5.dp),
+            colors = ButtonDefaults.buttonColors(
+                containerColor = Color.Transparent
+            ),
+            border = BorderStroke(1.dp, Color.LightGray)
+        ) {
+            Text("로그아웃", color = Color.LightGray)
```

Comment:
순서에 영향을 받지 않도록 'text = ' 부분을 추가해주시면 좋을 것 같습니다!

```suggestion
            Text(text = "로그아웃", color = Color.LightGray)
```